src/backend/.docker/Dockerfile

```diff
@@ -17,8 +17,11 @@ RUN npm run build
 RUN chown -R 1001:0 /usr/src/app && \
     chmod -R ug+rwx /usr/src/app
 
+# Provide access to node for adding logs
+RUN mkdir -p /.npm && chown -R 1001:0 /.npm && chmod -R ug+rwx /.npm
```

Comment:
Do we need to grant privilege to the user first before it can make a directory? Or does the order here not matter?

Reply:
Great question! This has occurred to me many many times, but I am not 100% on this! But since the unix system work with user Id (number) context - it assigns the user id to the directory; and while there is a need it checks if the provided user has access or not! That's my hunch, although I may be wrong here.

I hope that answers your question. Besides, the pipeline worked ✅